Stresser/RegistryEventHandler.cpp

```diff
@@ -6,9 +6,14 @@
 #include "../Utils/StringUtils.h"
 #include "../Utils/RegistryArtifactUtils.h"
 #include "../Utils/AutoCriticalSection.h"
+#include "../Utils/AutoSignedImageVerifier.h"
+#include "../Utils/LocalPcUtils.h"
 #include "../Utils/DebugPrint.h"
 #include "boost/algorithm/string.hpp"
 
+constexpr DWORD IdlePID = 0;
+constexpr DWORD SystemPID = 4;
+
```

Comment:
The same goes here: use upper case for constants.
Also, this is code duplication.

Better solution is to move this constants to `ProcessUtils` or `LocalPcUtils`